app/controllers/v0/mhv_opt_in_flags_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module V0
+  class MHVOptInFlagsController < ApplicationController
+    def show
+      opt_in_flag = MHVOptInFlag.find_by(user_account_id: current_user.account_uuid, feature: params[:feature])
```

Comment:
Trying to match the `Account` on the `User` to the `UserAccount` on `MHVOptInFlag`. As I mentioned with my other similar comment, should be: `MHVOptInFlag.find_by(user_account: current_user.user_account)`, will help catch bugs like this